app/js/components/orderEntry/ActiveOrders.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+class ActiveOrders extends React.Component {
+
+  render() {
+    return (
+      <div>
+        <br /> <br />
+        <div className="row">
+          <div className="col-lg-12 col-md-12 col-sm-12">
+            <h4> Active Orders </h4>
+            <div className="table-responsive">
+              <table className="table table-bordered">
+                <thead>
+                  <tr>
+                    <th className="text-center" style={{ width: "18%" }}>Date</th>
+                    <th className="text-center" style={{ width: "10%" }}>Status</th>
+                    <th className="text-center" style={{ width: "44%" }}>Details</th>
+                    <th className="text-center" style={{ width: "10%" }}>Action</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td className="text-center" style={{ fontSize: "14px" }}>
+                      started 12/03/2018
+                    </td>
+                    <td className="text-center" style={{ fontSize: "14px" }}>
+                      Active
+                    </td>
+                    <td style={{ paddingTop: "0" }}>
+                      <table className="table" style={{ marginBottom: "0" }}>
+                        <tbody>
+                          <tr style={{ fontSize: "14px" }}>
+                            <td> Oral </td>
```

Comment:
Did you intentionally hard code these?

Reply:
Yes I did, the components are just skeletons for the team to build on, so we can easily consume the Api and have consistent UI